### PR TITLE
Create discord.md

### PR DESCRIPTION
**Pull request by esparkhu**

### CODE REVIEW

File: content/discord.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@title discord
+@navbar
+
+# Polympics discord server
+
+https://discord.gg/kJG7eM4hDG
```
Comment (Artemis21) [inline content/discord.md]:
```suggestion
## [Polympics Discord Server](https://discord.gg/kJG7eM4hDG)

The official home of the event. Join to start chatting with fellow participants and team members!
```

Comment (Artemis21) [reply content/discord.md]: Note that there should never be more that one single-`#` on a page, Google will rank pages lower for it.

File: content/discord.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Offciala Battle of Polytopia main discord server
+
+https://discord.gg/polytopia
```
Comment (Artemis21) [inline content/discord.md]:
```suggestion
## [Official Polytopia Discord Server](https://discord.gg/polytopia)

The official Discord server for The Battle of Polytopia.
```

File: content/discord.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Note
+
+To register for the event please [login](login)
```
Comment (Artemis21) [inline content/discord.md]:
```suggestion
To register for the event please [/login](login).
```
(Absolute URL is required.)

**Review by Artemis21 — CHANGES_REQUESTED** [review]

**Review by esparkhu — COMMENTED** [review]
Okay. thanks

How do I approve the changes? Or do I need to?

**Comment by esparkhu** [comment]
Yes. Thanks.

**Comment by Artemis21** [comment]
There should be a "Commit suggestion" button?

**Comment by Artemis21** [comment]
> Yes. Thanks.

Was this meant to be in reply to something?

**Review by Artemis21 — APPROVED** [review]
Looks good

**Pull request merged**